src/orchestrations.js

```diff
@@ -40,7 +40,7 @@ exports.createOrchestration = (
   }
   if (requestParameters) {
     requestObject.queryString =
-      requestObject.queryString + querystring.stringify(requestParameters)
+      requestObject.queryString + '&' + querystring.stringify(requestParameters)
```

Comment:
@bradsawadye using `+` for string concatenation in JS is a bit controversial. It's not wrong, but it has the potential to cause bugs in future code you write when you're mixing concatenation of strings and numbers.
Since we're using ES6, Template literals would be the most consistent concatenation method seeing as they're used throughout the codebase.

```suggestion
      `${requestObject.queryString}&${querystring.stringify(requestParameters)}`
```